### PR TITLE
DOC6861 - Updated links

### PR DESCRIPTION
**Pull request by x8a**
Edited links to point to [New Relic data types](https://docs.newrelic.com/docs/telemetry-data-platform/understand-data/new-relic-data-types/)

### CODE REVIEW

File: src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudtrail-monitoring-integration.mdx [hunk 1]
```diff
@@ -35,7 +35,7 @@ You can change the polling frequency and filter data using [configuration option
 
 To find your integration data in Infrastructure, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > AWS** and select one of the AWS CloudTrail integration links.
 
-This integration does not provide metric or inventory data: only [event data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview). You can use Infrastructure's [**Events** page](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-events-page-live-feed-every-config-change) to view a timeline of these events.
+This integration does not provide metric or inventory data: only [event data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#events-new-relic). You can use Infrastructure's [**Events** page](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-events-page-live-feed-every-config-change) to view a timeline of these events.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudtrail-monitoring-integration.mdx:38]: Use relative link here.

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-app-engine-monitoring-integration.mdx [hunk 2]
```diff
@@ -32,7 +32,7 @@ New Relic queries your Google App Engine services based on a polling interval of
 
 ## Find and use data [#find-data]
 
-After [connecting the Google App Engine integration to New Relic](https://docs.newrelic.com/docs/connect-google-cloud-platform-services-infrastructure) and waiting a few minutes (based on the [polling frequency](#polling)), you can [find and use integration data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations):
+After [connecting the Google App Engine integration to New Relic](https://docs.newrelic.com/docs/connect-google-cloud-platform-services-infrastructure) and waiting a few minutes (based on the [polling frequency](#polling)), you can [find and use integration data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/):
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-app-engine-monitoring-integration.mdx:35]: Use relative link here.

File: src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-iot-monitoring-integration.mdx [hunk 3]
```diff
@@ -453,7 +453,7 @@ To view metric data for your AWS IoT integration, [create NRQL queries](/docs/in
 
 ## Inventory data [#inventory]
 
-To [view inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-in-infra) for AWS IoT, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+To [view inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for AWS IoT, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-iot-monitoring-integration.mdx:456]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-app-engine-monitoring-integration.mdx [hunk 4]
```diff
@@ -361,7 +361,7 @@ To view [metric data](/docs/integrations/new-relic-integrations/getting-started/
 
   ## Inventory data [#inventory]
 
-  To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for Google App Engine services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+  To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for Google App Engine services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-app-engine-monitoring-integration.mdx:364]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-bigquery-monitoring-integration.mdx [hunk 5]
```diff
@@ -231,7 +231,7 @@ Query `GcpBigQueryTableSample` events in Insights to view data for the following
 
 ## Inventory data
 
-To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for Google BigQuery services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following attributes.
+To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for Google BigQuery services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following attributes.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-bigquery-monitoring-integration.mdx:234]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-load-balancing-monitoring-integration.mdx [hunk 6]
```diff
@@ -300,7 +300,7 @@ Query `GcpInternalLoadBalancerSample` events in New Relic to view data for the f
 
 ## Inventory data [#inventory]
 
-To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for GCP Load Balancing services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for GCP Load Balancing services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-load-balancing-monitoring-integration.mdx:303]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-pubsub-monitoring-integration.mdx [hunk 7]
```diff
@@ -479,7 +479,7 @@ Query `GcpPubSubSubscriptionSample` events in New Relic to view data for the fol
 
 ## Inventory data [#rds-config]
 
-To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for GCP Pub/Sub services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for GCP Pub/Sub services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-pubsub-monitoring-integration.mdx:482]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-spanner-monitoring-integration.mdx [hunk 8]
```diff
@@ -177,7 +177,7 @@ Query `GcpSpannerDatabaseSample` events in New Relic to view data for the follow
 
 ## Inventory data
 
-To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for GCP Spanner services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for GCP Spanner services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-spanner-monitoring-integration.mdx:180]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-sql-monitoring-integration.mdx [hunk 9]
```diff
@@ -422,7 +422,7 @@ Query `GcpCloudSqlSample` events in New Relic to view data for the following att
 
 ## Inventory data
 
-To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#inventory-data) for GCP SQL services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
+To view [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview) for GCP SQL services, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > Inventory** and search for or select the following:
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-sql-monitoring-integration.mdx:425]: I think it's https://one.newrelic.com

File: src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-storage-monitoring-integration.mdx [hunk 10]
```diff
@@ -30,13 +30,13 @@ New Relic queries your Google Cloud Storage services based on a polling interval
 
 ## Find and use data [#find-data]
 
-After connecting the integration to New Relic and waiting a few minutes, data will appear in the New Relic UI. To [find and use integration data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations), including your dashboards and your alert settings, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > GCP > Google Cloud Storage**.
+After connecting the integration to New Relic and waiting a few minutes, data will appear in the New Relic UI. To [find and use integration data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/), including your dashboards and your alert settings, go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > GCP > Google Cloud Storage**.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/google-cloud-platform-integrations/gcp-integrations-list/google-cloud-storage-monitoring-integration.mdx:33]: I think it's https://one.newrelic.com

File: src/i18n/content/jp/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations.mdx [hunk 11]
```diff
@@ -219,7 +219,7 @@ Infrastructureインテグレーションは、[New Relicで使用](/docs/infras
         * イベントに添付された追加データ
         * メトリックスもしくはインベントリとは見なされないデータ
 
-        インテグレーションによっては、[その他のタイプの情報](https://docs.newrelic.co.jp/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#attributes)を属性として報告できます。
+        インテグレーションによっては、[その他のタイプの情報](/docs/telemetry-data-platform/understand-data/new-relic-data-types/)を属性として報告できます。
```
Comment (urbiz-nr) [inline src/i18n/content/jp/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations.mdx:222]: Not sure we update japanese docs ourselves, check with Julien!

Comment (x8a) [reply src/i18n/content/jp/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations.mdx:222]: I'll undo it!

**Pull request merged**